chatmaild/src/chatmaild/ini/chatmail.ini.f

```diff
@@ -23,8 +23,8 @@
 # days after which mails are unconditionally deleted
 delete_mails_after = 20
 
-# days after which users without a login are deleted (database and mails)
-delete_inactive_users_after = 100
+# days after which users without a successful login are deleted (database and mails)
+delete_inactive_users_after = 90
```

Comment:
We could change this only in https://github.com/deltachat/sysadmin-pyinfra/blob/main/servers/nine.testrun.org/chatmail.ini#L27, it doesn't need to be the new default IMHO - but no blocker.